cmd/start-ssh-agent.cmd

```diff
@@ -73,6 +73,8 @@ if exist "%GIT%" (
         set "HOME=%USERPROFILE%"
         "!SSH_ADD!"
         echo.
+    ) else (
+        set "HOME=%USERPROFILE%"

```

Comment:
True, that needs to be fixed. @c17r, could you please do this along the way of this PR (but in a separate commit)?